Documentation/ceph-object-multisite.md

```diff
@@ -165,12 +165,12 @@ kubectl create -f realm-a-keys.yaml
 
 Once the admin knows the endpoint and the secret for the keys has been created, the admin should create:
 
-1. A [CephObjectRealm](/design/ceph/object/realm.md) matching to the realm on the other Ceph cluster, with an endpoint as described above.
-1. A [CephObjectZoneGroup](/design/ceph/object/zone-group.md) matching the master zone group name or the master CephObjectZoneGroup from the cluster the the realm was pulled from.
-1. A [CephObjectZone](/design/ceph/object/zone.md) referring to the CephObjectZoneGroup created above.
-1. A [CephObjectStore](/design/ceph/object/store.md) referring to the new CephObjectZone resource.
+1. A [CephObjectRealm](https://github.com/rook/rook/blob/master/design/ceph/object/realm.md) matching to the realm on the other Ceph cluster, with an endpoint as described above.
+1. A [CephObjectZoneGroup](https://github.com/rook/rook/blob/master/design/ceph/object/zone-group.md) matching the master zone group name or the master CephObjectZoneGroup from the cluster the the realm was pulled from.
+1. A [CephObjectZone](https://github.com/rook/rook/blob/master/design/ceph/object/zone.md) referring to the CephObjectZoneGroup created above.
+1. A [CephObjectStore](https://github.com/rook/rook/blob/master/design/ceph/object/store.md) referring to the new CephObjectZone resource.
```

Comment:
Should these point to the CRD docs instead of design docs? Thoughts @travisn?

Reply:
Oh good point, the docs should only link to other docs in the Documentation folder. The designs are targeted for developers instead of admins. I'd suggest we go ahead and just fix these links to keep this PR simple, then separately review the docs to see how to rearrange the topic and avoid linking to the design docs.